principles.md

```diff
@@ -0,0 +1,65 @@
+# Fejlesztési alapelvek
+## Együttműködés
+Mindenki külön branchen dolgozik.
+
+Feature branch név formátum: feature/{feature neve}/{fejlesztő neve}, pl. feature/auth/den.

```

Comment:
Szerintem itt a fejlesztő neve alapból felesleges, ha valaki szeretne valamit javítani a feature-ön, akkor hozzá tudja adni a nevét (vagy valami mást, ami leírja, hogy mit javít), hogy meg legyen különböztetve a két ág. Ez azért is jó, mert egyszerűbb látni, hogy melyik a "main" feature branch, amibe a sajátot majd merge-elni kell.

Reply:
https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

Ennel tobb fixen nem kell

Reply:
Commit message formát nem akarom erőltetni. Egyrészt minek, annyit nem ad mint amennyi gondolatot kéne belefektetni, másrészt az anyázás sokkal viccesebb :D

Reply:
> Szerintem itt a fejlesztő neve alapból felesleges

Valóban, fő feature branchre tényleg nem kell.